src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  people: Person[]
+  selectedSlug: string;
+}
+
+export const PeopleTable: React.FC<Props> = ({ people, selectedSlug }) => {
+  const findParent = (
+    ArrayOfPeople: Person[],
```

Comment:
Consider renaming the 'ArrayOfPeople' parameter to 'arrayOfPeople' for consistency with camelCase naming conventions 🐫